src/phpDocumentor/Configuration/PathNormalizingMiddleware.php

```diff
@@ -87,6 +87,10 @@ private function makeDsnRelativeToConfig(Configuration $configuration, UriInterf
 
         /** @var array{name: string, location?: ?Path, parameters?: array<string, mixed>} $template */
         foreach ($configuration['phpdocumentor']['templates'] as $key => $template) {
+            if (!isset($template['location'])) {
```

Comment:
I have a personal preference to read the if statements more verbose. the `!` is easy to overlook, not specifically to in this case but in other cases it is. So please change this to:

```suggestion
            if (isset($template['location']) === false) {
```

Apart from that, thanks for solving this issue!